fence/blueprints/login/google.py

```diff
@@ -11,6 +11,12 @@ def get(self):
         flask.redirect_url = flask.request.args.get("redirect")
         if flask.redirect_url:
             flask.session["redirect"] = flask.redirect_url
+        if flask.current_app.config.get("MOCK_GOOGLE_AUTH", False) == True:
```

Comment:
don't need `== True`. it will default to False. 
```suggestion
        if flask.current_app.config.get("MOCK_GOOGLE_AUTH", False):
```

Reply:
Fixed in 6fcaa8d0f5374b85bd9fb13710e92c81b53f2bc7